src/test/java/tests/TokenTests.java

```diff
@@ -122,4 +121,47 @@ public Object[][] DatasetWithTokenIdInvalid(){
                 {" "}
         };
     }
+    @Test(description = "get token txs | success", groups = {"token"})
+    public void getTokenTxs(){
+        tokenSteps.getTokenTxs("asset1d9v7aptfvpx7we2la8f25kwprkj2ma5rp6uwzv")
+                .validateStatusCode(HttpURLConnection.HTTP_OK);
+    }
+    @Test(description = "get token txs invalid page", groups = {"token"}, dataProvider = "paramTokenTxsPageInvalid")
+    public void getTokenTxsInvalidPage(String page, int size){
+        Map<String, Object> param = new CreateParameters()
+                .withPage(page)
+                .withPageSize(size)
+                .getParamsMap();
+        tokenSteps.getTokenTxsParamInvalid("asset1d9v7aptfvpx7we2la8f25kwprkj2ma5rp6uwzv", param)
+                .validateResponse(HttpURLConnection.HTTP_OK);
+    }
+    @DataProvider(name = "paramTokenTxsPageInvalid")
+    public Object[][] DatasetTokenTxsPageInvalid(){
+        return new Object[][]{
+                {" ", 5},
+                {"", 5},
+                {null, 5},
+                {"abc", 5},
+                {"@#$", 5}
+        };
+    }
+    @Test(description = "get token txs invalid size", groups = {"token"}, dataProvider = "paramTokenTxsSizeInvalid")
```

Comment:
expect of this @Test same as @Test above, so create this @Test is unnecessary, should be add dataProvider into @Test above